terraform/e2e/main.tf

```diff
@@ -59,3 +59,34 @@ module "policy_service" {
   oidc_service_account = module.common_infra.oidc_service_account
   gcs_events_filter    = var.policy_gcs_events_filter
 }
+
+resource "random_id" "default" {
```

Comment:
Why do you need this for e2e env?

Reply:
I thought a random prefix can't do any harm. So you prefer to just keep the name as `pmap-static-dev-bucket`?

Reply:
I mean why you need to add all these resources like storage bucket in e2e env

Reply:
I got it. You want to test the customized scope test case  in the `pmap-dev`, right?Please add a comment to explain what all these resources are for and later clean the static bucket in `pmap-ci` with another PR.

Reply:
Because we want the e2e test to get resource from another project to make sure the customize resource scope functionality is working. So we create a static bucket in pamp-dev project.

Reply:
I don't think e2e module is for this purpose, it is for us and customers to set up infrastructure required for pmap, it does not make sense to create ci resource in this module. 

I wonder why we cannot test custom scope in pmap-ci?

Reply:
Because for most resources, they have `projects/{project_id}` within its name, and we can parse that from the name and use it as the scope. However, GCS bucket's name is special, it doesn't have `projects/{project_id}` in its name. So when getting info from GCS buckets, the search scope is defined by `defaultScope`. And if user didn't define a `defaultScope`, `defaultScope` will be set to `project/{project_id}`, To make sure `defaultScope` is working when set to `folders` or `organization`, we need to test on some resources that's not from the same project.

Reply:
Please correct me, I thought we don't set the default anymore and users need to provide one?

Reply:
This should help? https://github.com/abcxyz/pmap/pull/142/files#r1259018960

Reply:
+1 We don't set the default anymore and users need to provide one, right? `And if user didn't define a defaultScope, defaultScope will be set to project/{project_id}`, this logic is not correct.

Reply:
From the discussion, I am thinking only create a non-GCS resource in ci project, and use existing pmap-dev GCS bucket to test `defaultResouceScope`.

Reply:
Agree with @sqin2019 that the e2e module is not for this purpose. The test bucket should only be created in the ci module.